lfr/7pin_cardinal_demo/README

```diff
@@ -0,0 +1,12 @@
+Users can unzip the nekRS mesh file and IC file 
```

Comment:
Add that they need to use LFS to get the files first?
Also spell out IC

Reply:
Correct, I have updated the REAMDE.